ng-dev/utils/BUILD.bazel

```diff
@@ -9,8 +9,8 @@ ts_library(
 
     # prodmode target must be set to es5 as the Octokit class, which is extended is not a proper
     # prototyped object.
-    prodmode_target = "es5",
-    visibility = ["//dev-infra:__subpackages__"],
+    #prodmode_target = "es5",
```

Comment:
Did you test something out here? should this be re-added or what's the plan?

Reply:
We have since moved away from the problematic Octokit class with our upgrade of `@octokit/*`

Reply:
Ah nice! we should remove the comment then. 👍